### PR TITLE
Fix modules code quote documentations 

### LINKED ISSUE #63516: Module docs code quotes must use uppercase C
##### SUMMARY
The module documentation uses the syntax `C(code snippet)` for short code quotations. When a lower-case `c` is used by mistake, the content does not display correctly.

Find and fix instances of `c(code snippet)` in the module documentation. For example: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/google/gcp_cloudbuild_trigger.py#L327.

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME
docs.ansible.com

##### ANSIBLE VERSION
2.10

##### CONFIGURATION
N/A

##### OS / ENVIRONMENT
N/A

### PR DESCRIPTION
**Pull request by Hyde46**
##### SUMMARY
Replace erroneous lower case c in documentation string as module docs code quotes must use uppercase C. 
Fixes #63516

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com

### CODE REVIEW

File: lib/ansible/modules/cloud/google/gcp_compute_snapshot.py [hunk 1]
```diff
@@ -167,9 +167,9 @@
 notes:
 - 'API Reference: U(https://cloud.google.com/compute/docs/reference/rest/v1/snapshots)'
 - 'Official Documentation: U(https://cloud.google.com/compute/docs/disks/create-snapshots)'
-- for authentication, you can set service_account_file using the c(gcp_service_account_file)
+- for authentication, you can set service_account_file using the C(gcp_service_account_file)
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/google/gcp_compute_snapshot.py:170]: `gcp_*` modules are auto-generated and these changes will be clobbered when this file is regenerated. You might want to fix these changes [here](https://github.com/GoogleCloudPlatform/magic-modules/blob/2aea12555416b1d4300e95c4085cec432997a41c/provider/ansible/documentation.rb#L171)

**Review by Akasurde — CHANGES_REQUESTED** [review]
Thanks for the contribution.

**Comment by Akasurde** [comment]
@Hyde46 There are several files missing here which matches the criteria - 

```
./lib/ansible/modules/network/f5/bigip_provision.py
./lib/ansible/modules/network/f5/bigip_gtm_topology_region.py
./lib/ansible/modules/network/f5/bigip_gtm_topology_record.py
./lib/ansible/modules/network/f5/bigip_asm_policy_fetch.py
./lib/ansible/modules/clustering/k8s/_k8s_raw.py
./lib/ansible/modules/clustering/k8s/k8s.py
./lib/ansible/modules/cloud/amazon/rds_instance.py
./lib/ansible/modules/cloud/heroku/heroku_collaborator.py
./lib/ansible/modules/cloud/azure/azure_rm_appgateway.py
./lib/ansible/modules/windows/win_iis_webbinding.py
./lib/ansible/modules/windows/win_dns_record.py
```

File: lib/ansible/modules/cloud/google/gcp_compute_snapshot.py (hunk 1 above)
Comment (Hyde46) [reply lib/ansible/modules/cloud/google/gcp_compute_snapshot.py:170]: Good point, will fix it!

**Comment by Hyde46** [comment]
Ill include these files in the next commit. Thanks

**Comment by Hyde46** [comment]
Fixed the missing files.

File: lib/ansible/modules/cloud/google/gcp_compute_snapshot.py (hunk 1 above)
Comment (Hyde46) [reply lib/ansible/modules/cloud/google/gcp_compute_snapshot.py:170]: Fixed, see [here](https://github.com/GoogleCloudPlatform/magic-modules/pull/2490)

**Review by Hyde46 — COMMENTED** [review]
Fixes lower c character, replace with upper case C

**Comment by samccann** [comment]
Hi @Hyde46. Can you please rebase this PR?  Instructions are here - https://docs.ansible.com/ansible/latest/dev_guide/developing_rebasing.html

Thanks!

**Comment by samccann** [comment]
Sorry if you saw the mistaken rebase comment, that's not required. Reviewing now

**Review by samccann — APPROVED** [review]

**Pull request merged**